src/getters/getRoomMetadata.ts

```diff
@@ -0,0 +1,5 @@
+import { RoomMetadata } from '../utils/types'
+
+export default function getRoomMetadata(description?: string): RoomMetadata {
+    return JSON.parse(description!)
+}
```

Comment:
We got `description?: string` and immediately after we tell ts `description!`. Maybe it could go w/o `?` and `!` and we make sure `description` IS there whenever we call `getRoomMetadata`.